From: Szymon Wilczek <swilczek.lx@gmail.com>
Subject: mailmap: update email address for Szymon Wilczek
Date: Sun, 21 Dec 2025 16:17:10 +0100

Map my old address <szymonwilczek@gmx.com> to my new address
<swilczek.lx@gmail.com>.  The old account is no longer accessible due to
provider blocking access.

Link: https://lkml.kernel.org/r/20251221151710.13747-1-swilczek.lx@gmail.com
Signed-off-by: Szymon Wilczek <swilczek.lx@gmail.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 .mailmap |    1 +
 1 file changed, 1 insertion(+)

--- a/.mailmap~mailmap-update-email-address-for-szymon-wilczek
+++ a/.mailmap
@@ -794,6 +794,7 @@ Sven Eckelmann <sven@narfation.org> <sve
 Sven Eckelmann <sven@narfation.org> <sven.eckelmann@openmesh.com>
 Sven Eckelmann <sven@narfation.org> <sven@open-mesh.com>
 Sven Peter <sven@kernel.org> <sven@svenpeter.dev>
+Szymon Wilczek <swilczek.lx@gmail.com> <szymonwilczek@gmx.com>
 Takashi YOSHII <takashi.yoshii.zj@renesas.com>
 Tamizh Chelvam Raja <quic_tamizhr@quicinc.com> <tamizhr@codeaurora.org>
 Taniya Das <quic_tdas@quicinc.com> <tdas@codeaurora.org>
_
